amd_openvx_extensions/amd_rpp/source/Downmix.cpp

```diff
@@ -0,0 +1,271 @@
+/*
+Copyright (c) 2019 - 2022 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Please change the copyright from 2022 to 2023

Reply:
Changed it